code/web/src/modules/profileForm/ProfileForm.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import Button from '../../ui/button';
+import { saveProfile } from '../user/api/actions';
+import { changeEditMode } from '../user/api/actions';
+import Availability from '../availability/Availability';
+
+
+// import { updateAvailability } from '../subscription/api/actions';
+
+class ProfileForm extends Component{
+  constructor(props) {
+    super(props)
+
+      this.state = {
+        img: '',
+        address: '',
+        email: '',
+        description: ''
+      }
+  }
+
+  updateProfileState(e) {
+    this.setState({ [e.target.name]: e.target.value })
+  }
+
+  updateProfile = () => {
+    let newState = {
+          img: this.state.img || this.props.user.details.img || '',
+          address: this.state.address || this.props.user.details.address || '',
+          email: this.state.email || this.props.user.details.email || '',
+          description: this.state.description || this.props.user.details.description || ''
+        }
+    this.props.saveProfile(newState)
+    this.props.changeEditMode(this.props.user)
+  }
+
+  render() {
+      return (
+        <section style={{ display: 'flex', justifyContent: 'center' }}>
+        {this.props.user.isEditMode ?
+          (
+            <article>  
+              {/* look into image file/upload */}
+              <label htmlFor='img-input'>
```

Comment:
We should definitely break out each `< input />` with its attributes into multiple lines to make it more readable.

Reply:
I agree! I think there are quite a few things we can break out on multiple lines to clean it up a bit.